pkgs/os-specific/linux/kernel/generate-config.pl

```diff
@@ -28,7 +28,7 @@
 while (<ANSWERS>) {
     chomp;
     s/#.*//;
-    if (/^\s*([A-Za-z0-9_]+)(\?)?\s+(\S+)\s*$/) {
+    if (/^\s*([A-Za-z0-9_]+)(\?)?\s+(.*\S)\s*$/) {
```

Comment:
do we still need the \S then ?

Reply:
Without the terminating `\S`, the `.*` will match trailing whitespace. The original pattern ignored leading and trailing whitespace and I didn't want to change that behavior.